components/partials/spot/orderbook.vue

```diff
@@ -44,12 +54,14 @@ const components = {
 
 export default Vue.extend({
   components: {
+    'v-aggregation-selector': AggregationSelector,
     'v-trades': Trades,
     'v-orderbook': Orderbook
   },
 
   data() {
     return {
+      aggregation: 3, // default aggregation decimal
```

Comment:
Great job! The only thing I'd like you to change is to set the default aggregation price decimals to the `market.priceDecimals` value.